aiodocker/docker.py

```diff
@@ -355,12 +379,16 @@ def __init__(self, docker, **kwargs):
         }
         params.update(kwargs)
 
+        inspect_info = await self.show()
+
+        isTty = inspect_info['Config']['Tty']
```

Comment:
Please rename this variable and argument of `multiplexed_result` to `is_tty` in favor of [PEP-8 coding style](https://www.python.org/dev/peps/pep-0008/).